game/game_controller.rb

```diff
@@ -53,6 +54,14 @@ def active_player
     end
   end
 
+  def inactive_player
+    if board.empty_spaces.odd?
```

Comment:
When making a simple if else like this you can use the : ? ternary operator. 
```

def inactive_player 
  board.empty_spaces.odd? ? players[1] : players[2]
end

```
https://en.wikipedia.org/wiki/%3F:#Ruby